src/config/config.php

```diff
@@ -50,10 +48,9 @@
     |   persist_id_token        (Boolean) Optional. Indicates if you want to persist the id token, default false
     |
     */
-
-    // 'persist_user' => true,
-    // 'persist_access_token' => true,
-    // 'persist_id_token' => true,
+    'persist_user' => true,
+    'persist_access_token' => false,
```

Comment:
We don't want to persist the access token by default?

Reply:
I don't see a good reason to ... regardless, the default is false, as per the documentation above, so just matching that in the example config.

Reply:
That's fine, only saw the `// 'persist_access_token' => true,` in the diff.